docs/_includes/head.html

```diff
@@ -14,6 +14,13 @@
     {% else %}
         <title>{{ site.title }}</title>
     {% endif %}
+    <!-- Google Tag Manager -->
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-NL48VKT');</script>
```

Comment:
There is already another gtag.js include further down in this file on line 37, is this supposed to go alongside of that, or should it replace it instead?

The include further down also has a conditional to only include it for production builds, not sure if that's wanted/necessary here?

Reply:
I double checked with the SEO team, the initial idea was to add a second container, but they confirmed they can reuse the existing container. I'm going to close out this request. Thanks!